ca/certificate-authority_test.go

```diff
@@ -585,14 +612,14 @@ func TestRejectValidityTooLong(t *testing.T) {
 	// Test that the CA rejects CSRs that would expire after the intermediate cert
 	csrDER, _ := hex.DecodeString(NoCNCSRhex)
 	csr, _ := x509.ParseCertificateRequest(csrDER)
-	_, err = ca.IssueCertificate(*csr, 1, FarPast)
+	_, err = ca.IssueCertificate(*csr, 1, "bogus", FarPast)
```

Comment:
It might be worth a comment to note that negative tests (looking for issuance to fail) don't need to `awaitIssuance()`, since if the CA agrees to issue, by returning successfully from `IssueCertificate()`, then it has already failed.